nuts_and_bolts/templates/layout.html

```diff
@@ -57,7 +58,11 @@
   <footer class="footer bg-dark">
     <div class="container nav-font text-center">
       <a href="{{ url_for('main.contact_us') }}">Contact Us</a>
-    </div>
+    </div>     
+
+    <div class="container nav-font text-center">
+      <a href="{{ url_for('main.faq') }}">FAQ</a>
+    </div>     
```

Comment:
Put this link in the same div as the "Contact Us" link above it.

Reply:
when I was doing this It did not look very good visually. Not that it looked great that way